person.rb

```diff
@@ -0,0 +1,20 @@
+def initialize(id, name: 'Unknown', age: 0, parent_permission: true)
+  @id = id
+  @name = name
+  @age = age
+  @parent_permission = parent_permission
+end
+
+attr_writer :name
+
+attr_writer :age
```

Comment:
- [ ] According to the project requirement we should create a getter and setter for the variables `@name` and `@age` and only a getter for `@id`.  Therefore I suggest you add them to follow the project requirement.
```rb
  attr_accessor :name, :age
  attr_reader :id
```